alcs-frontend/src/app/shared/meeting-overview/meeting-overview.component.ts

```diff
@@ -12,7 +12,7 @@ import { UserService } from '../../services/user/user.service';
 
 type MeetingWithApplications = {
   meetingDate: number;
-  applications: (UpcomingMeetingDto & { isExpanded: boolean; isHighlighted: boolean })[];
```

Comment:
just to double check, is it still expands and scrolls to meeting using the search feature

Reply:
yep! It's just that applications themselves can't be expanded anymore so there is no need to keep a isExpanded boolean for them.